src/coreclr/jit/lowerxarch.cpp

```diff
@@ -10642,6 +10642,8 @@ void Lowering::ContainCheckHWIntrinsic(GenTreeHWIntrinsic* node)
 
                             case NI_BMI2_MultiplyNoFlags:
                             case NI_BMI2_X64_MultiplyNoFlags:
+                            case NI_X86Base_Multiply:
+                            case NI_X86Base_X64_Multiply:
```

Comment:
I am mostly guessing that the "containment" check and switching of operations should be the same, and the resulting code looks ok, but I am not sure